assets/shaders/octree/leafBorderTransfer.comp.glsl

```diff
@@ -15,6 +15,17 @@ uniform uint octreeLevel;
 #include "./_brickCoordinates.glsl"
 #include "./_threadNodeUtil.glsl"
 
+vec4 calculateBorderFinalValue(vec4 borderValue, vec4 neighborBorderValue) {
```

Comment:
During one of my tests, it looked better without treating border nodes differently in the border transfer, but we'll see later

Reply:
Totally agree